packages/amplify_storage_s3/test/amplify_storage_s3_downloadfile_test.dart

```diff
@@ -48,46 +46,23 @@ void main() {
   });
 
   test(
-      'Throws StorageException when method channel result does not include a file path',
+      'downloadFile throws a StorageException',
```

Comment:
More descriptive test names are easier to understand without reading the code.
eg. `A PlatformException results in a StorageException being thrown`

Reply:
Fixed.